_docs/11-training.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: splash
+title: "Training"
+permalink: /docs/training/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+
+# Training Material
+## Below is a list of training available on Youtube.
+
+* [FPGA](https://www.youtube.com/watch?v=8cXOZ-V31kA&list=PL9B4edd-p2ajpeOI4k4Lr_fwXcRlwO3dv)
+* [PolarFire SoC - Debugging a Bare Metal Application](https://www.youtube.com/watch?v=F4_wR3e62og&list=PL9B4edd-p2ahc3AQdBAmGfAyo_pcDW0Tg)
+* [PolarFire SoC - Bare Metal Applications from the LIM](https://www.youtube.com/watch?v=iMWq_Ue5DbU&list=PL9B4edd-p2ahGOmnvJvFLvSID3N3-rJC6)
+* [PolarFire SoC - Bare Metal Applications from the DDR](https://www.youtube.com/watch?v=pOMvi_uTauU&list=PL9B4edd-p2ai1Z0E78PDV9gf6EnBX3B19)
+* [PolarFire SoC - Building a Yocto Image](https://www.youtube.com/watch?v=CweRIou2yxk&list=PL9B4edd-p2agS6QkXUZjAvjlPiyAO5RP0)
+* [PolarFire SoC - Bare Metal Applications Accessing the Fabric](https://www.youtube.com/watch?v=xsQyJEcdEQI&list=PL9B4edd-p2aiVmuKT_cpkODBdhXNDYA1w)
+* [PolarFire SoC - Introduction to GPIO/UIO Drivers and Yocto SDK](https://www.youtube.com/watch?v=VT5oON9JLSI&list=PL9B4edd-p2aiesb5K9TKQpoyGJlPAmq8x)
+* [PolarFire SoC - Github introduction](https://www.youtube.com/watch?v=-e_OniBC5nU&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=4)
+* [PolarFire SoC - Bare Metal Interrupts](https://www.youtube.com/watch?v=DOL7GBTKOoA&list=PL9B4edd-p2ahg_MASDPEwOy8spzsL5_f5)
+* [PolarFire SoC - Generating a Design for the PolarFire SoC Icicle Kit with Custom IP](https://www.youtube.com/watch?v=AGmJ7XOjYAM&list=PL9B4edd-p2ahUAL56N_8FuIS3BJFzApEF)
+* [PolarFire SoC - Getting back to factory defaults](https://www.youtube.com/watch?v=dL3u6pYWFvQ&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=5)
+* [PolarFire SoC - Factory MSS configuration for the Icicle Kit]( https://www.youtube.com/watch?v=mXp5cNg4qbY&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=6)
+* [PolarFire SoC - What is the HSS](https://www.youtube.com/watch?v=UKGVAl3BOog&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=8)
+* [PolarFire SoC - MSS Configurator Run through](https://www.youtube.com/watch?v=WrkmaJingRg&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=7)
+
+## Webinars
+*  [Renode Webinar Series](https://www.microsemi.com/product-directory/fpga-soc/5210-mi-v-embedded-ecosystem#renode-webinar-series)
```

Comment:
Change the display text from "Renode Webinar Series"
to
Getting started with PolarFire SoC - Renode Webinar Series
or
Introduction to PolarFire SoC - Renode Webinar Series

I like the "Getting started ..." option

Reply:
Added the Getting Started ... option to the Renode Webinar Series.